client/app/lib/util/cleanup.coffee

```diff
@@ -38,3 +77,9 @@ module.exports = ->
     for option in options
       if kookies.expire key, option
         console.log "#{key} deleted"
+
+  for exp in regexWith
```

Comment:
thought it should work too but when I tested, the engine I used returned false, so I made it fail safe. but as you say it works, we can keep as is.